server/service/src/login.rs

```diff
@@ -132,17 +131,13 @@ impl LoginService {
                 });
             }
         };
+        let service_ctx = service_provider.context("", &user_account.id.clone())?;
```

Comment:
Something really hacky to get the user id.... 👀

Reply:
or `let mut service_ctx` and then set the user_id directly on the original ctx (still not super nice but then you don't open up a new DB connection...).

Reply:
I think @clemens-msupply  meant;

service_ctx.user_id = ...